client/src/components/EventDetailsModal.jsx

```diff
@@ -1,31 +1,34 @@
 import { useState } from 'react';
 import { Status } from "../utils/utils";
-import { useUser } from "../contexts/UserContext";
 import "../styles/Modal.css";
 
 const EventDetailsModal = ( {onModalClose, eventData, initialStatus, onStatusChange}) => {
 
-    const [statusState, setStatusState] = useState(initialStatus); //change to event object? 
+    const [statusState, setStatusState] = useState(initialStatus); 
```

Comment:
Nit: consider moving this to Event since we have UI that needs to rerender if this value changes as well.